Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/controllers/CaseController.java

```diff
@@ -55,6 +56,13 @@
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
 
+    @RequestMapping(value="/suit/{suitId}/removeCases", method = RequestMethod.POST, consumes = "application/json")
```

Comment:
Let's change it to RequestMethod.DELETE and remove 'removeCases'. Should be something like /suit/{suitId}/cases

Reply:
In vue.js when we send Delete request we can't add body for request, but we need there, so we create POST method.
The name of the method I'll change.